src/load_data.jl

```diff
@@ -4,10 +4,7 @@ asset_path(args...) = joinpath(PKG_DIR, "assets", args...)
 """
     load_data(name)
 
-Load a dataset as a `DataFrame`.
-
-All available datasets can be found in the [`assets/`](https://github.com/xKDR/Survey.jl/tree/main/assets)
-directory.
+Load a sample dataset provided in the [`assets/`](https://github.com/xKDR/Survey.jl/tree/main/assets) directory a `DataFrame`.
```

Comment:
I think the previous docstring was better for the same reasons I mentioned above. The first sentence should be short and concise. Extra details are given separately.

Reply:
The latter is only 7 characters more.